test/docfx.Test/DocfxTest.cs

```diff
@@ -90,6 +90,11 @@ private static (string docsetPath, string appDataPath, string outputPath, Dictio
             var cachePath = Path.Combine(appDataPath, "cache");
             var statePath = Path.Combine(appDataPath, "state");
 
+            if (spec.Inputs.ContainsKey("repositoryUrl"))
+            {
+                Environment.SetEnvironmentVariable("DOCFX_REPOSITORY_URL", spec.Inputs["repositoryUrl"]);
```

Comment:
Setting environment variables here will cause mult-threading problems as env var is a global shared resource and we run all tests in parallel.

If you want to mock a repo in spec test, search `repos:` in yaml files